opendata-python/tests/test_models.py

```diff
@@ -112,3 +112,10 @@ def test_store_locally(self, remote_athlete, local_storage):
             os.listdir(os.path.join(local_storage.strpath, settings.data_prefix))
         assert f'{{{remote_athlete.id}}}.json' in \
             os.listdir(os.path.join(local_storage.strpath, settings.metadata_prefix))
+
+    def test_store_locally_no_data(self, remote_athlete, local_storage):
+            remote_athlete.store_locally(data=False)
+            assert remote_athlete.id in \
```

Comment:
Shouldn't this be `assert ... not in ...`?

Reply:
yes, there were eventually more test that needed to be changed